### DISCUSSION TITLE
Github Account Hacked

### DISCUSSION THREAD

**Discussion by jsBrutha** [community/community#137519] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi everyone,

I need some urgent help. It seems that my account has been compromised, and I can no longer log in. As a result, I also can't access my Copilot subscription, which is really disrupting my work.

Has anyone else experienced this? What steps should I take to regain control of my account and restore access to my Copilot subscription? Any advice or guidance would be greatly appreciated.

Thanks in advance for your help!

**Comment by ThisIs-Developer** [comment]
Hi, I had a similar issue a couple of months ago. My account was suspended, and I couldn’t log in either. I opened a ticket on GitHub (compile), which didn’t require a login. Unfortunately, I received no response from GitHub for quite some time. I even reached out to some individuals online who claimed they could help, but they were just scammers asking for money.

After about three months, GitHub's support team finally responded. They only needed to verify my email address, and then my account was restored. 

So, in your case, I recommend being patient and waiting for GitHub support to get back to you. You can’t do much beyond this, and it's best to avoid any third-party services that might offer to fix the issue for a fee.

Hang in there, and I hope your issue gets resolved soon!

**Comment by ebndev** [comment]
Hi @jsBrutha,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
